ironrdp-session/src/lib.rs

```diff
@@ -38,4 +42,5 @@ pub struct InputConfig {
     pub global_channel_name: String,
     pub user_channel_name: String,
     pub graphics_config: Option<GraphicsConfig>,
+    pub gfx_dump_file: Option<PathBuf>,
```

Comment:
This sounds like a downstream option that should be handled in `ironrdp-client`’s `Arg` struct directly rather than `ironrdp-session`’s `InputConfig`. What do you think?

Reply:
Agreed. Missed this in refactoring.